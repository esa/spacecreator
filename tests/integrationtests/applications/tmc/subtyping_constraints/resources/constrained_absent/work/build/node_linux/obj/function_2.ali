V "GNAT Lib v8"
A -gnatwa
A -O0
A -Wall
A -Wextra
A -gnatA
A -fdata-sections
A -ffunction-sections
A -gnatd.E
A -fstack-usage
A -g
A -fstack-check=specific
A -mtune=generic
A -march=x86-64
P ZX

RN
RV NO_UNCHECKED_CONVERSION
RV NO_IMPLEMENTATION_ATTRIBUTES
RV NO_IMPLEMENTATION_PRAGMAS
RV NO_ELABORATION_CODE
RV SPARK_05

U function_2%b		function_2.adb		5aafcdee OO PK
W ada%s			ada.ads			ada.ali
W ada.numerics%s	a-numeri.ads		a-numeri.ali
W ada.numerics.generic_elementary_functions%s
W ada.unchecked_conversion%s
Z interfaces%s		interfac.ads		interfac.ali
W system%s		system.ads		system.ali
W system.io%s		s-io.adb		s-io.ali

U function_2%s		function_2.ads		d9dd7fdf EB EE NE OO PK
W ada%s			ada.ads			ada.ali
W ada.characters%s	a-charac.ads		a-charac.ali
W ada.characters.handling%s  a-chahan.adb	a-chahan.ali
W adaasn1rtl%s		adaasn1rtl.adb		adaasn1rtl.ali
W constrained_absent_dataview%s  constrained_absent_dataview.adb  constrained_absent_dataview.ali
W function_2_datamodel%s  function_2_datamodel.adb  function_2_datamodel.ali
W function_2_ri%s	function_2_ri.adb	function_2_ri.ali
W interfaces%s		interfac.ads		interfac.ali
W interfaces.c%s	i-c.adb			i-c.ali
W interfaces.c.strings%s  i-cstrin.adb		i-cstrin.ali
Z system%s		system.ads		system.ali
Z system.img_enum_new%s	s-imenne.adb		s-imenne.ali
W system_dataview%s	system_dataview.adb	system_dataview.ali
W taste_basictypes%s	taste_basictypes.adb	taste_basictypes.ali

D ada.ads		20170911084841 76789da1 ada%s
D a-charac.ads		20170908151217 2d3ec45b ada.characters%s
D a-chahan.ads		20180111085525 1601a50f ada.characters.handling%s
D a-chlat1.ads		20170908151217 66457d31 ada.characters.latin_1%s
D a-numeri.ads		20170908151217 bb51c45a ada.numerics%s
D a-numaux.ads		20180111085525 243fe0b7 ada.numerics.aux%s
D a-ngelfu.ads		20180111085525 cb378f71 ada.numerics.generic_elementary_functions%s
D a-ngelfu.adb		20180111085525 a9022048 ada.numerics.generic_elementary_functions%b
D a-unccon.ads		20170908151217 0e9b276f ada.unchecked_conversion%s
D adaasn1rtl.ads	20230103105717 ba1d55d4 adaasn1rtl%s
D board_config.ads	20230103105717 ff248fc1 board_config%s
D constrained_absent_dataview.ads  20230103105717 ecfc98a1 constrained_absent_dataview%s
D function_2.ads	20230103105716 3d8cebb8 function_2%s
D function_2.adb	20230103105716 3ccd7907 function_2%b
D function_2_datamodel.ads  20230103105717 c13a4ea0 function_2_datamodel%s
D function_2_ri.ads	20230103105703 11029cea function_2_ri%s
D interfac.ads		20180111085525 5ab55268 interfaces%s
D i-c.ads		20170908151217 5421119b interfaces.c%s
D i-cstrin.ads		20180111085525 341d0ddf interfaces.c.strings%s
D system.ads		20180111085525 4635ec04 system%s
D s-bitops.ads		20180111085525 0765e3a3 system.bit_ops%s
D s-exctab.ads		20180111085525 54135002 system.exception_table%s
D s-imenne.ads		20180111085525 614c06c0 system.img_enum_new%s
D s-io.ads		20180111085525 9e42f74f system.io%s
D s-parame.ads		20180111085525 f896c45c system.parameters%s
D s-stalib.ads		20180111085525 09bd3940 system.standard_library%s
D s-unstyp.ads		20180111085525 34867c83 system.unsigned_types%s
D system_dataview.ads	20230103105717 0a0f8283 system_dataview%s
D taste_basictypes.ads	20230103105717 102ef363 taste_basictypes%s
X 1 ada.ads
16K9*Ada 20e8 13|5r6 9r5 14|6r6 7r6
X 2 a-charac.ads
16K13*Characters 18e19 13|5r10 9r9
X 3 a-chahan.ads
36K24*Handling 159e28 13|5w21 9r20
X 5 a-numeri.ads
16K13*Numerics 32e17 14|7r10
X 7 a-ngelfu.ads
39k22*Generic_Elementary_Functions 205e46 14|7w19
X 9 a-unccon.ads
20v14*Unchecked_Conversion 14|6w10
X 10 adaasn1rtl.ads
5K9*adaasn1rtl 125e15 13|17w6 18r5
19B12*Asn1Boolean{boolean}
X 12 constrained_absent_dataview.ads
26K9*CONSTRAINED_ABSENT_DATAVIEW 134e32 13|11w6 12r5 26r42
70E6*asn1SccMyChoice_selection 70e57 13|26r70
75R6*asn1SccMyChoice 83e11 13|33r37
115R6*asn1SccMySeqOf 118e11 13|30r30 14|10r30
X 13 function_2.ads
22K9*Function_2 36l5 36e15 14|9b14 48l5 48t15
23r4*ctxt{15|43R6} 27r104 14|12r15 27m19 44m10
26V13*To_Mychoice_selection{15|58E6} 26>36
26e36 Src{12|70E6} 26r234
27V13*Get_State{19|47P9}
28U14*Startup 14|41b14 45l11 45t18 47s7
30U14*PI_1 30=19 31i<c,function_2_PI_PI_1>21 14|10b14 16l11 16t15
30r19 p1{12|115R6} 14|10b19
33U14*RI_0_PI_2=33:76
33r25 p1{12|75R6}
34U14*Execute_Transition 34>34 14|14s16 19b14 38l11 38t29 43s10
34i34 Id{integer} 14|19b34 20r25
35N4*CS_Only 14|14r36 29r21
X 14 function_2.adb
20i7 trId{integer} 22r17 23r18 26m19 30m19
35L15 Continuous_Signals 28r24 31r24
36L15 Next_Transition
X 15 function_2_datamodel.ads
26K9*Function_2_Datamodel 13|19w6 19r32 26r104 26r166 15|75e25
31E6*asn1SccFunction_2_States 13|27r72 15|31e48
31n35*asn1Sccwait{31E6} 14|27r33
43R6*asn1SccFunction_2_Context 13|23r19 15|46e11
44e5*state{31E6} 13|27r109 14|12r20 27m24
45b5*init_done{10|19E12} 13|24m8 14|44m15
58E6*asn1SccFunction_2_Mychoice_Selection 13|26r125 26r187 15|58e83
X 16 function_2_ri.ads
9K9*Function_2_RI 13|21w6 27r44 33r62 16|19e18
11U14*Pi_2 13|33r76
17V13*To_C_Pointer{19|47P9} 13|27s58
X 17 interfac.ads
38K9*Interfaces 13|3w6 4r6 7r5 8r5 17|184e15
X 18 i-c.ads
18K20*C 13|4r17 8r16 18|230e17
X 19 i-cstrin.ads
36K22*Strings 13|4w19 8r18 19|106e25
47P9*chars_ptr 13|27r30
X 20 system.ads
37K9*System 14|3r6 4r5 20|156e11
X 24 s-io.ads
37K16*IO 14|3w13 4r12 24|64e14
X 28 system_dataview.ads
26K9*System_Dataview 13|15w6 16r5 28|48e20
X 29 taste_basictypes.ads
26K9*TASTE_BasicTypes 13|13w6 14r5 29|95e21

