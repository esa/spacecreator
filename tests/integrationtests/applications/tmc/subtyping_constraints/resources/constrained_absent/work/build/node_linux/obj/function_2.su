function_2.ads:26:4:To_Mychoice_Selection	32	static
function_2.ads:27:4:Get_State	96	static
function_2.adb:10:4:Pi_1	32	static
function_2.adb:19:4:Execute_Transition	16	static
function_2.adb:41:4:Startup	16	static
function_2.adb:9:1:Function_2	16	static
