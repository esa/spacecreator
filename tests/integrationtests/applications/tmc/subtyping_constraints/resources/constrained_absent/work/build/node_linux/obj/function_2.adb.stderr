function_2.adb:3:06: warning: "System.IO" is an internal GNAT unit
function_2.adb:3:06: warning: use of this unit is non-portable and version-dependent
function_2.adb:3:12: warning: no entities of "System.IO" are referenced
function_2.adb:4:01: warning: use clause for package "IO" has no effect
function_2.adb:7:18: warning: unit "Ada.Numerics.Generic_Elementary_Functions" is not referenced
function_2.adb:10:19: warning: formal parameter "p1" is not referenced
function_2.adb:36:15: warning: label "Next_Transition" is not referenced
function_2.ads:5:20: warning: no entities of "Ada.Characters.Handling" are referenced
function_2.ads:7:01: warning: use clause for package "Handling" has no effect
function_2.ads:7:01: warning: use clause for package "Interfaces" has no effect
function_2.ads:13:06: warning: no entities of "TASTE_BasicTypes" are referenced
function_2.ads:14:01: warning: use clause for package "TASTE_BasicTypes" has no effect
function_2.ads:15:06: warning: no entities of "System_Dataview" are referenced
function_2.ads:16:01: warning: use clause for package "System_Dataview" has no effect
function_2.ads:17:06: warning: no entities of "adaasn1rtl" are referenced
function_2.ads:18:01: warning: use clause for package "adaasn1rtl" has no effect
